.idea/src/lesson17/Task17/CollEctions.java

```diff
@@ -0,0 +1,76 @@
+package lesson17.Task17;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+class CollEctions {
+    public static void main(String[] args) {
+
+
+        Integer[] numberArray = new Integer[14];
+        final Integer[] temp = {0};
+
+        for (int i = 0; i < numberArray.length; i++) {
+            numberArray[i] = (int) (Math.random() * 10);
+        }
+
+        class Test1 implements Iterator {
```

Comment:
Where is your Iterator interface?
It should be like Iterator from `java.lutil` 
method `hasNext` must return boolean value whether our collection has the next element or not.
and method next must return the next element from the collection